metricbeat/module/elasticsearch/elasticsearch.go

```diff
@@ -613,12 +612,12 @@ func getSettingGroup(allSettings mapstr.M, groupKey string) (mapstr.M, error) {
 
 	settings, err := allSettings.GetValue(groupKey)
 	if err != nil {
-		return nil, errors.Wrap(err, "failure to extract "+groupKey+" settings")
+		return nil, fmt.Errorf("failure to extract "+groupKey+" settings"+": %w", err)
 	}
 
 	v, ok := settings.(map[string]interface{})
 	if !ok {
-		return nil, errors.Wrap(err, groupKey+" settings are not a map")
+		return nil, fmt.Errorf(groupKey+" settings are not a map"+": %w", err)
```

Comment:
same here, potential nil?

Reply:
@cmacknz

Reply:
We must have a nil error here since a non-nil error was caught at 614.

Reply:
fixed